flexget/plugins/sites/lostfilm.py

```diff
@@ -47,151 +38,124 @@ class LostFilm(object):
-                pass
-
-            episode_id = 'S{:02d}E{:02d}'.format(
-                season_num, episode_num)
+                raise PluginError('Cannot parse new releases page')
 
-            for item in page.findAll('div', 'inner-box--item'):
-                torrent_link = quality = None
+            # Iterate over releases
+            seen_buttons = index_page.find_all('div', 'haveseen-btn')
+            for seen_btn in seen_buttons:
                 try:
-                    torrent_link = item.find(
-                        'div', 'inner-box--link sub').a['href']
-                    quality = item.find('div', 'inner-box--label').text.strip()
-                except:
-                    log.debug('Item doesn\'t have a link or quality')
-                    continue
-                if torrent_link is None or quality is None:
-                    log.debug('Item doesn\'t have a link or quality')
+                    details_pane = seen_btn.parent.find('div', 'details-pane')
+                    episode_name_rus = details_pane.find('div', 'alpha').text.strip()
+                    episode_name_eng = details_pane.find('div', 'beta').text.strip()
+                    series_name_rus = details_pane.parent.find('div', 'name-ru').text.strip()
+                    series_name_eng = details_pane.parent.find('div', 'name-en').text.strip()
+                    lostfilm_num, season_num, episode_num = [int(x) for x in seen_btn['data-code'].split('-')]
+                except AttributeError:
+                    log.exception('Could not parse %s, probably HTML markup was changed', release_url)
                     continue
-                if quality_map.get(quality):
-                    quality, file_ext = quality_map.get(quality)
-                else:
-                    file_ext = 'avi'
-                if series_name_eng:
-                    new_title = '.'.join(
-                        [
-                            series_name_eng,
-                            episode_id,
-                            quality,
-                            'rus.LostFilm.TV',
-                            file_ext,
-                            'torrent'
-                        ]
-                    ).replace(' ', '.')
-                else:
-                    if item.get('title') is not None:
-                        new_title = '{} {}'.format(
-                            item['title'],
-                            quality)
-                    else:
-                        log.debug('Item doesn\'t have a title')
-                        continue
-                new_entry = Entry()
-                new_entry['url'] = torrent_link
-                new_entry['title'] = new_title.strip()
-                if series_name_rus:
+
+                # TODO We cannot determine available qualities right now since they are listed on the download page.
+                # So, assume all possible qualities are available and emit entries for them
```

Comment:
Available qualities are not listed on the releases page.

Reply:
Guess you have to do it the way you did before. I didn't realise quality wasn't listed in the search results.

Reply:
So I need to revert the _urlrewriter_ stuff?

Reply:
I guess so, sorry...

Reply:
@cvium done, please check